parameter/FloatingPointParameterType.cpp

```diff
@@ -172,9 +174,7 @@ bool CFloatingPointParameterType::toBlackboard(
         }
 
         // Move to the "raw memory" value space
-        // Using an intermediary reference variable to avoid klocwork false positive
-        const uint32_t &value = reinterpret_cast<const uint32_t &>(fValue);
-        uiValue = value;
+        utility::binaryCopy(fValue, uiValue);
```

Comment:
~~~cpp
uiValue = utility::binaryCopy<decltype(uiValue>(fValue)
~~~
Type can be autodeduce.